auth/auth.go

```diff
@@ -0,0 +1,149 @@
+package auth
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"fmt"
+	"io/ioutil"
+	"crypto/rsa"
+	"time"
+
+	"github.com/DarthHater/bored-board-service/database"
+	"github.com/DarthHater/bored-board-service/model"
+	"github.com/DarthHater/bored-board-service/constants"
+	"github.com/gin-gonic/gin"
+	"github.com/dgrijalva/jwt-go"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	signKey   *rsa.PrivateKey
+	verifyKey *rsa.PublicKey
+)
+
+// ReadAndSetKeys will read public and private RSA keys and create a key for signing JWTs.
+func ReadAndSetKeys() {
+	privKeyPath := os.Getenv("PRIVATE_KEY_PATH")
+	pubKeyPath := os.Getenv("PUBLIC_KEY_PATH")
+
+	signBytes, err := ioutil.ReadFile(privKeyPath)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
You might do:

```
log.WithFields(log.Fields{
"privateKeyPath": privKeyPath,
}).Fatal(err)
```

This will give the private Key Path as part of what gets logged so you can see if it's set to something wonky.